spec/error_spec.rb

```diff
@@ -4,13 +4,14 @@ module CoverMyMeds
   module Error
     describe HTTPError do
 
-      subject { described_class.new status, error_json, http_method, rest_resource }
+      subject { described_class.new status, error_json, http_method, url }
 
       specify "#message" do
-        expect(subject.message).to eq(
-          "#{status}: #{error_json}\n"+
-          "in response to:\n"+
-          "#{http_method.upcase} #{rest_resource}\n"
+        expect(subject.message).to eq(<<-EOS
```

Comment:
All this did before was reproduce the exact implementation.  That is kind of like testing that a == a, and really shouldn't ever fail, regardless of how we *want* the output to look.